### ISSUE TITLE
can't register an account on pinterest (the horror.)

### ISSUE THREAD

**Issue by CouldBeThis** [anonaddy/anonaddy#30]
Finally I can make an account on pinterest (whoever is doing SEO for them is so good it comes up as a top hit on so many searches of mine! ever though I hate it). I never wanted to before because I assume it's really just a marketing company and probably doing all sorts of creepy stuff. Anyway I thought with anonaddy I could make an account (or maybe a bunch of them). But alas, no!

I have a paid annonaddy account so tried a few combos of the UUID, the words, and a couple of the different available domains as well as a couple of custom subdomains. Sometimes it seems to work and I am redirected to login but the credentials don't let me in. Sometimes it reloads with the error "Your email is invalid" or something like "that doesn't look like an email address" (I can't seem to duplicate that right now) or "Ooops! Sorry! Something went wrong on our end." The different error messages show up differently (some only after submitting, some in a fake popup dialogue, some while entering the info) which suggests (to a person who has no idea how these things work) that there is more than one blocking mechanism. (Or pinterest likes to mix things up?)

In no case do I get a verification email which is standard for account creation with all services. 

I don't know if the domains themselves are blocked or there is something about the structure of the addresses that it doesn't like. I also tried using @user.33mail.com it didn't work either.

Any idea as to a workaround? 

Not at all urgent. I have lived this long without pinterest so I will continue to do so. More posting because if there is an issue with them it might be more widespread, or become moreso as anonaddy is more established and well known.

**Comment by willbrowningme** [comment]
I just tried this myself and it doesn't show any error but when inspecting the requests made it did fail with error "Invalid email".

Perhaps Pinterest are blocking the server's IP as I also tried using a custom domain and that didn't work either.

Strange since I know there are some users on AnonAddy who have told me they've signed up for Pinterest previously.

I cannot think of any work around at the moment unfortunately, I will try contacting them to see what the issue is.
